hack/make/test-docker-py

```diff
@@ -3,18 +3,29 @@ set -e
 
 source hack/make/.integration-test-helpers
 
-# subshell so that we can export PATH without breaking other things
+# The commit or tag to use for testing
+# TODO docker 17.06 cli client used in CI fails to build using a sha;
+# docker build -t docker-sdk-python3:ac922192959870774ad8428344d9faa0555f7ba6 -f Dockerfile-py3 https://github.com/docker/docker-py.git#ac922192959870774ad8428344d9faa0555f7ba6
+# unable to prepare context: unable to 'git clone' to temporary context directory: error fetching: error: no such remote ref ac922192959870774ad8428344d9faa0555f7ba6
+#: exit status 128
+#DOCKER_PY_COMMIT=${DOCKER_PY_COMMIT:-ac922192959870774ad8428344d9faa0555f7ba6}
+DOCKER_PY_COMMIT="${DOCKER_PY_COMMIT:-3.7.0}"
+
 (
 	bundle .integration-daemon-start
+	if [ -z "${DOCKER_SOCK}" ]; then
```

Comment:
perhaps instead, we can check if `DOCKER_HOST` starts with `unix://` and strip that

Reply:
Did even better; test can now run against remote daemons as well (which could be localhost); I changed the script to start the container with `--network=host` if `DOCKER_HOST` uses `tcp://` 🎉